src/Microsoft.DotNet.Build.Tasks/PackageFiles/PerfTesting.targets

```diff
@@ -1,123 +1,67 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-
-  <!-- Perf Runner NuGet package paths -->
-  <PropertyGroup Condition="'$(TargetOS)'=='Linux'">
-    <PerfDotNetCliDir>$PACKAGE_DIR/perf.dotnetcli</PerfDotNetCliDir>
-    <PerfDotNetCliInstaller>$(ToolsDir)/RunnerScripts/xunitrunner-perf/ubuntu-dotnet-local-install.sh</PerfDotNetCliInstaller>
-    <XunitPerfRunnerPackageId>Microsoft.DotNet.xunit.performance.runner.cli</XunitPerfRunnerPackageId>
-  </PropertyGroup>
-  <!-- Perf Runner NuGet package paths -->
-  <PropertyGroup Condition="'$(TargetOS)'=='Windows_NT'">
-    <XunitPerfRunnerPackageId>Microsoft.DotNet.xunit.performance.runner.Windows</XunitPerfRunnerPackageId>
-  </PropertyGroup>
-
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!-- Perf Analysis NuGet package paths -->
-  <PropertyGroup Condition="'$(TargetOS)'=='Linux'">
+  <PropertyGroup>
     <XunitPerfAnalysisPackageId>Microsoft.DotNet.xunit.performance.analysis.cli</XunitPerfAnalysisPackageId>
     <XunitPerfAnalysisPath>$(PackagesDir)$(XunitPerfAnalysisPackageId)/$(XunitPerfAnalysisPackageVersion)/lib/netstandard1.3/Microsoft.DotNet.xunit.performance.analysis.cli.dll</XunitPerfAnalysisPath>
+	<TraceEventPackage>Microsoft.Diagnostics.Tracing.TraceEvent\1.0.0-alpha-experimental</TraceEventPackage>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(TargetOS)'=='Windows_NT'">
-    <XunitPerfAnalysisPackageId>Microsoft.DotNet.xunit.performance.analysis</XunitPerfAnalysisPackageId>
-    <XunitPerfAnalysisPath>$(PackagesDir)$(XunitPerfAnalysisPackageId)/$(XunitPerfAnalysisPackageVersion)/tools/xunit.performance.analysis.exe</XunitPerfAnalysisPath>
-    <XunitPerfAnalysisDir>$(PackagesDir)$(XunitPerfAnalysisPackageId)/$(XunitPerfAnalysisPackageVersion)/tools</XunitPerfAnalysisDir>
-  </PropertyGroup>
-
-
-  <!-- Installing dotnet cli version that can execute the xunit perf test runner -->
-  <Target Name="InstallDotnetCliForPerfExecution" Condition="'$(TargetOS)'=='Linux' AND '$(Performance)'=='true' AND !Exists('$(PerfDotNetCliDir)/dotnet') AND '$(OsEnvironment)'=='Unix'" AfterTargets="CopyXunitExecutionFiles" >
-    <Exec Command="chmod +x $(PerfDotNetCliInstaller)" />
-    <Exec Command="$(PerfDotNetCliInstaller) -d $(PackagesDir)/perf.dotnetcli -v $(PackagesDir)$(XunitPerfAnalysisPackageId)/$(XunitPerfAnalysisPackageVersion)/lib/netstandard1.3/DotNetCliVersion.txt" ContinueOnError="true"/>
+  
+  <ItemGroup>
+	<TraceEventNativePath Include="$(PackagesDir)\$(TraceEventPackage)\lib\native\**\*.*" />
+  </ItemGroup>
+  
+  <Target Name ="PublishPerfRunner" Condition="'$(Performance)'=='true'" BeforeTargets="RunTestsForProject">
```

Comment:
I think we can solve this is two parts.  The first is moving the PerfRunner to work like RemoteExecutorConsoleApp.  This will remove the bulk of the goo that we are doing here.  We will still however need to copy over the native portions of TraceEvent, the last step in this target.  The way that the code currently works we need to have the native folders in the directory of where the TraceEvent code is being executed ( @brianrob or @vancem please correct me if I am wrong.